### PR TITLE
Use accounting.js to format the custom donation amount. Closes #50.

### LINKED ISSUE #50: Use accounting.js to format the custom donation
I use [accounting.js](http://openexchangerates.github.io/accounting.js/) in WooCommerce Name Your Price to format the user-entered value on the front-end. I think it could be useful here too. The money format settings are already in the admin, so it should be pretty straight-forward.

### PR DESCRIPTION
**Pull request by helgatheviking**
Needs to be tested against different currency formats. Also, does charitable handle the formatting on processing?

### CODE REVIEW

**Comment by ericnicolaas** [comment]
@helgatheviking This is looking good. The only change I'd like to see is to have the `get_currency_format()` method in the `Charitable_Public` class moved into the `Charitable_Currency` class instead, and renamed to `get_accounting_js_format()`.

**Comment by helgatheviking** [comment]
That should do it.

File: includes/currency/class-charitable-currency.php [hunk 1]
```diff
@@ -175,6 +175,37 @@ public function get_currency_format() {
...
+                $format = "%s %v";
+            break;
+            case 'right-with-space':
+                $format = "%v %s";
+            break;
+            default:
+                $format = "%s%v";
+            break;
+        }
+        $format = "%s%v";
```
Comment (ericnicolaas) [inline includes/currency/class-charitable-currency.php]: @helgatheviking Resetting `$format` here overrides the earlier switch. If you can remove this line it's good to be merged. :)

**Comment by ericnicolaas** [comment]
Merged in 8f68fdd87aee4092b20633a74e10f423dd44f188

**Pull request closed without merging**